caasp-kvm/caasp-kvm

```
@@ -48,27 +50,28 @@ Usage:
 
   * Local git checkouts
 
-     --salt-dir <DIR>      the Salt repo checkout (Default: CAASP_SALT_DIR)
-     --manifests-dir <DIR> the manifests repo checkout (Default: CAASP_MANIFESTS_DIR)
-     --velum-dir <DIR>     the Velum repo checkout (Default: CAASP_VELUM_DIR)
+     --salt-dir <DIR>           the Salt repo checkout (Default: CAASP_SALT_DIR)
+     --manifests-dir <DIR>      the manifests repo checkout (Default: CAASP_MANIFESTS_DIR)
+     --velum-dir <DIR>          the Velum repo checkout (Default: CAASP_VELUM_DIR)
 
   * Advanced Options
 
-    --admin-ram <INT>      RAM to allocate to admin node (Default: CAASP_ADMIN_RAM=$ADMIN_RAM)
-    --admin-cpu <INT>      CPUs to allocate to admin node (Default: CAASP_ADMIN_CPU=$ADMIN_CPU)
-    --master-ram <INT>     RAM to allocate to master node(s) (Default: CAASP_MASTER_RAM=$MASTER_RAM)
-    --worker-ram <INT>     CPUs to allocate to master node(s) (Default: CAASP_MASTER_CPU=$MASTER_CPU)
-    --master-cpu <INT>     RAM to allocate to worker node(s) (Default: CAASP_WORKER_RAM=$WORKER_RAM)
-    --worker-cpu <INT>     CPUs to allocate to worker node(s) (Default: CAASP_WORKER_CPU=$WORKER_CPU)
+    --admin-ram <INT>           RAM to allocate to admin node (Default: CAASP_ADMIN_RAM=$ADMIN_RAM)
+    --admin-cpu <INT>           CPUs to allocate to admin node (Default: CAASP_ADMIN_CPU=$ADMIN_CPU)
+    --master-ram <INT>          RAM to allocate to master node(s) (Default: CAASP_MASTER_RAM=$MASTER_RAM)
+    --worker-ram <INT>          CPUs to allocate to master node(s) (Default: CAASP_MASTER_CPU=$MASTER_CPU)
+    --master-cpu <INT>          RAM to allocate to worker node(s) (Default: CAASP_WORKER_RAM=$WORKER_RAM)
+    --worker-cpu <INT>          CPUs to allocate to worker node(s) (Default: CAASP_WORKER_CPU=$WORKER_CPU)
+    --extra-repo <STR>          URL of a custom repository on the master(s)/worker(s) (Default: CAASP_EXTRA_REPO=$($DIR/../misc-tools/download-image --expand-channel --type kvm channel://devel))
```

Is the default not extra repos? This seems misleading :)